src/openrct2/scenario/Scenario.cpp

```diff
@@ -713,13 +710,13 @@ void scenario_remove_trackless_rides(rct_s6_data* s6)
 
 static void scenario_objective_check_guests_by()
```

Comment:
Are you planning on making these member functions?

Reply:
Eventually, yes.